app/server/sonicpi/test/test_tick.rb

```diff
@@ -118,6 +118,15 @@ def test_keyed_ticks
       assert_equal(0, tick(:baz))
     end
 
+    def test_string_keyed_ticks
+      tick_reset_all
+      assert_equal(0, tick("foo"))
```

Comment:
Even though ticks start at 0, they're still 0 after the first tick. That way you can use tick to iterate through a ring:

````
live_loop :foo do
  play (scale :e3, :minor).tick, release: 0.1
  sleep 0.125
end
```

This will play *all* the notes in the scale. If the first tick returned `1` as you suggest, we'd never hear the first note :-)